pkg/cmd/flags/capture.go

```diff
@@ -71,16 +70,20 @@ func PrepareCapture(captureSlice []string) (tracee.CaptureConfig, error) {
 		} else if cap == "mem" {
 			capture.Mem = true
 		} else if cap == "network" || cap == "net" {
-			capture.Net.CaptureProcess = true
+			// default capture mode: a single pcap file with all traffic
+			capture.Net.CaptureSingle = true
 		} else if strings.HasPrefix(cap, "pcap:") {
-			// TODO: turn default capture mode to a single pcap file like it was
-			capture.Net.CaptureProcess = true // process is default capture mode
+			found := false
+			capture.Net.CaptureSingle = true // single is set even without --capture net
 			context := strings.TrimPrefix(cap, "pcap:")
 			fields := strings.Split(context, ",")
-			found := false
 			for _, field := range fields {
+				capture.Net.CaptureSingle = false // remove default single pcap mode
+				if field == "single" {
+					found = true // single was requested, enable it at the end
```

Comment:
Why do we need this found variable? Can't we simply set `capture.Net.CaptureSingle = true` here?

Reply:
This is so if the user executes:

```
tracee-ebpf --capture pcap:XXX
```

without the `--capture network` argument, it will activate the single mode by default, like if `--capture network` argument was given, and only remove the mode if it hasn't been chosen in `XXX`.

But the logic is wrong =D, fixing it.

Reply:
Re-checking the logic and testing it.